pkg/argsbuilder/submit_pytorchjob.go

```diff
@@ -72,6 +72,7 @@ func (s *SubmitPytorchJobArgsBuilder) AddCommandFlags(command *cobra.Command) {
 	command.Flags().StringVar(&s.args.CleanPodPolicy, "clean-task-policy", "None", "How to clean tasks after Training is done, support None, Running, All.")
 	command.Flags().StringVar(&s.args.Cpu, "cpu", "", "the cpu resource to use for the training, like 1 for 1 core.")
 	command.Flags().StringVar(&s.args.Memory, "memory", "", "the memory resource to use for the training, like 1Gi.")
+	command.Flags().Int64Var(&s.args.ActiveDeadlineSeconds, "active-deadline-seconds", 0, "Specifies the duration (in seconds) since startTime during which the job can remain active before it is terminated")
```

Comment:
I think  the default value 0 for active-deadline-seconds will mislead the user. How about not setting default value? If it's not setting,  There is no ActiveDeadlineSeconds.

Reply:
Done, If it's not setting, There is no ActiveDeadlineSeconds in job's CR.